tests/test_basic.py

```diff
@@ -58,7 +58,7 @@ def test_pipeline():
 
     Ms, Mu = adata.layers["Ms"][0], adata.layers["Mu"][0]
     Vs, Vd = adata.layers["velocity"][0], adata.layers["dynamical_velocity"][0]
-    Vpca, Vgraph = adata.obsm["velocity_pca"][0], adata.uns["velocity_graph"].data[:5]
+    _, Vgraph = adata.obsm["velocity_pca"][0], adata.uns["velocity_graph"].data[:5]
```

Comment:
just remove `adata.obsm["velocity_pca"][0]` (I think I originally wanted to have another assertion here).